charts/redpanda/values.yaml

```
@@ -931,6 +931,8 @@ listeners:
   admin:
     # -- The port for both internal and external connections to the Admin API.
     port: 9644
+    # -- Optional instrumentation hint - https://kubernetes.io/docs/concepts/services-networking/service/#application-protocol
+    # appProtocol: "http"
```

It appears there's a weird quirk of `helm-docs` where it doesn't really handle commented out lines. You'll notice that `README.md` now states that `"http"` is the default value which isn't true.

```suggestion
    # appProtocol:
```